pkg/server/serverpb/status.proto

```diff
@@ -399,11 +399,18 @@ message RangeProblems {
 message RangeStatistics {
   // Queries per second served by this range.
   //
-  // Note that queries per second will only be known by the leaseholder.
-  // All other replicas will report it as 0.
+  // Batch requests per second served by this range.
   double queries_per_second = 1;
   // Writes per second served by this range.
   double writes_per_second = 2;
+  // Requests per second served by this range.
+  double requests_per_second = 3;
```

Comment:
A bit of commentary talking about the relation between requests_per_second, writes_per_second and reads_per_seconds would help (even if trivial).

Reply:
Good idea. Updated.